rebar.config

```diff
@@ -1,17 +1,38 @@
-{deps, []}.
+{deps, [
+    {erlang_lorawan, ".*",
+        {git, "https://github.com/helium/erlang-lorawan.git", {branch, "master"}}},
+    {grpc_client,
+        {git, "https://github.com/Bluehouse-Technology/grpc_client.git", {branch, "master"}}},
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git", {branch, "master"}}},
+    {helium_proto, {git, "https://github.com/helium/proto.git", {branch, "master"}}},
+    {lager, "3.9.2"}
+]}.
 
 {erl_opts, [
     debug_info,
+    {parse_transform, lager_transform},
     warnings_as_errors
 ]}.
 
+{grpc, [
+    {protos, "protos"},
+    {gpb_opts, [{module_name_suffix, "_pb"}]}
+]}.
+
 {plugins, [
+    {grpcbox_plugin,
+        {git, "https://github.com/novalabsxyz/grpcbox_plugin.git", {branch, "master"}}},
     {rebar3_format, "1.2.1"},
     {erlfmt, "1.0.0"}
 ]}.
 
+{pre_hooks, [
+    {compile, "make grpc"},
```

Comment:
Is there a way we can not have this pre_hook?
It recompiles grpc when you do anything with rebar3.

Reply:
Yes, there is a way.  It was already accounted for via `Makefile` such that this `pre_hook` was redundant.

e.g., within Makefile: `compile: | $(grpc_services_directory)`

Fixed in commit 5531bd9.